### ISSUE TITLE
Attributes set into initial bind seems to disappear

### ISSUE THREAD

**Issue by pdonorio** [alecthomas/flask_injector#20]
Hi, I'm using injector, flask and flask-restful classes.

When I bind my object `obj` I set some models with meta syntax like:
```
setattr(obj, 'ModelName', MyModelClass)
# then
binder.bind(ObjClass, to=obj, scope=self.singleton)
```

If I check for this `obj.ModelName` before requests, they are there and fine.
When I recover with the injector inside the Flask-Restful init function the object, the model is not there anymore. 

Unluckily my case is more complicated than this, but I wanted to ask first if this is a possible normal behaviour and why. If not I may build a small app to replicate.

Thank you a lot for this cool project!

**Comment by jstasiak** [comment]
Hi, my first impression here would be that this is not expected, do you mind providing a minimal test case to reproduce this?

**Comment by pdonorio** [comment]
Hi again,
I want to clarify that my app wants to inject a set of own flask extensions made by me of various services/db into any Flask-Restful class. Thereby to provide the minimal case I started an empty flask app with the simplest view that requires a flask extension following the README. And it didn't work :/.

I then noticed that there has been some changes since I started using the project (probably because of the previous issue I commented on :) ) kinda month ago. So I discovered that the latest modification on the README, see commit edade0616f730e2a089e6b62d2aadf1699d4da1 , is what make things not work. The singleton/provides does not launch any configure at flask init time, so calling the injection would fail with:
```
  File "/usr/local/lib/python3.6/dist-packages/injector.py", line 782, in create_object
    init(instance, **additional_kwargs)
injector.CallError: Call to ExtClass.__init__() failed: __init__() missing 1 required positional argument: 'app' (injection stack: [<class '__main__.Waz'>])
```

Here is a gist to help you in testing what I am saying, probably there is a bug to fix:
https://gist.github.com/pdonorio/5b18fcc86d040f4f13cea3f649fea2a8

Tell me if I can help you in any way on this problem.
